template.json

```diff
@@ -40,7 +41,8 @@
       "lint:eslint": "eslint ./src --ext .js,.jsx,.ts,.tsx --cache",
       "fix": "npm-run-all fix:*",
       "fix:eslint": "npm run lint:eslint -- --fix",
-      "fix:prettier": "prettier \"src/**/*.js,jsx,ts,tsx,scss}\" --write"
+      "fix:prettier": "prettier \"src/**/*.js,jsx,ts,tsx,scss}\" --write",
+      "deploy": "cross-env rsync -avPx build/* ${npm_package_name}@<host>:~/public/ --delete-after"
```

Comment:
will `<host>` automatically be replace or is that a manual process?

Reply:
it's a manual process, because this is open source, we can't just add the monkapps host. Even if we did, we have different monkapps regions that would have to be configured here as well.

Reply:
@ThijsTyZ see the [README](https://github.com/mediamonks/cra-template/pull/7/files#diff-04c6e90faac2675aa89e2176d2eec7d8) from this PR, now you can define the **host** in your **package.json**